recipes-aos/aos-messageproxy/aos-messageproxy_git.bb

```diff
@@ -18,51 +15,26 @@ SRC_URI += " \
     file://aos-cm-service.conf \
 "
 
-FILES:${PN} += " \
-    ${sysconfdir} \
-    ${systemd_system_unitdir} \
-"
-
-inherit go goarch systemd
-
-SYSTEMD_SERVICE:${PN} = "aos-messageproxy.service aos-messageproxy-provisioning.service"
-
-DEPENDS = "xen-tools"
-RDEPENDS:${PN} = "xen-tools-libxenvchan"
-
-# embed version
-GO_LDFLAGS += '-ldflags="-X main.GitSummary=`git --git-dir=${S}/src/${GO_IMPORT}/.git describe --tags --always`"'
-
-# WA to support go install for v 1.18
+DEPENDS += "poco systemd grpc grpc-native protobuf-native protobuf openssl-native curl"
```

Comment:
Why openssl-native curl are required?

Reply:
because we use openssl for handshake, and curl for download

Reply:
Then it should be RDEPENDS? no? Is it required for build?